samples/tutorial/step6.cr

```diff
@@ -89,12 +89,14 @@ if File.exists? HISTFILE # Does it exist?
 end
 
 begin # Get rid of stacktrace on ^C
+	fancy.tty.switch_to_alternate_screen
```

Comment:
The indention is off, please check if your editor is set to use two spaces per level.  Most added lines suffer from this.

Reply:
My mistake, I prefer tabs but I've set up a commit hook to remove them in future.